portal_pages/templates/portal_pages/includes/contact.html

```diff
@@ -0,0 +1,44 @@
+
+<div class="third"><div class="cushion">
+
+    <h2>Contact</h2>
+
+    {% if contact.address_1 %}
+        <p>{{ contact.address_1 }}</p>
+    {% endif %}
+
+    {% if contact.address_2 %}
+        <p>{{ contact.address_2 }}</p>
+    {% endif %}
+
+    <p>
+        {% if contact.city %}
```

Comment:
Also when there's no labels or formatting being added by the conditional block...it's not really necessary as just {{ contact.city }} in this case will render nothing if there is nothing there...